pkg/resolver/searchSchema.go

```diff
@@ -0,0 +1,83 @@
+package resolver
+
+import (
+	"context"
+
+	"github.com/doug-martin/goqu/v9"
+	"github.com/driftprogramming/pgxpoolmock"
+	db "github.com/stolostron/search-v2-api/pkg/database"
+	klog "k8s.io/klog/v2"
+)
+
+type SearchSchemaMessage struct {
+	pool   pgxpoolmock.PgxPool
+	query  string
+	params []interface{}
+}
+
+func SearchSchema(ctx context.Context) (map[string]interface{}, error) {
+	searchSchemaResult := &SearchSchemaMessage{
+		pool: db.GetConnection(),
+	}
+	searchSchemaResult.searchSchemaQuery(ctx)
+	return searchSchemaResult.searchSchemaResults(ctx)
+}
+
+func (s *SearchSchemaMessage) searchSchemaQuery(ctx context.Context) {
```

Comment:
For consistency with search.go
```suggestion
// Build the query to get all the properties (or keys) from the resources in the database.
// These are used to build the search schema.
func (s *SearchSchemaMessage) buildSearchSchemaQuery(ctx context.Context) {
```